src/metabase/lib/metadata/calculation.cljc

```diff
@@ -92,6 +94,9 @@
                          :stage-number stage-number}
                         e)))))))
 
+(defmethod display-name-method :dispatch-type/nil [_query _stage-number _x _stage]
+  "")
```

Comment:
This seems like kind of an icky hack. Shouldn't the display name of `nil` be `nil`?

Reply:
Removed.